css/dashboard.css

```diff
@@ -0,0 +1,51 @@
+.dashboard {
+  flex: 0 0 800px;
+  align-content: center;
+  flex-flow: row wrap;
+  /* The top margin places the dashboard below the tabstrip, the bottom margin
+  visually centers the tiles vertically. We need the bottom margin because we're
+  using flexbox to align the tiles, but the tiles themselves are not visually
+  even, since they have extra space below for the title. */
+  margin: 100px auto 30px;
+  width: 800px;
+}
```

Comment:
@paulrouget Let's do the responsiveness in a separate update. Rather than reflowing tiles, I want to physically shrink the tiles using a media query so the grid is always 4x3.

It is important that the grid always stay the same if we plan to use it for bookmarks.